docs/usage.md

```diff
@@ -123,6 +123,16 @@ for common RNA-seq library preparation kits.
 | `--pico`  | SMARTer Stranded Total RNA-Seq Kit - Pico Input | 3     | 0     | 0     | 3     |
 
 
+### strand direction for StringTie and feturecounts
+The strandedness of the library can be set py using the `--forward_stranded`  and `--reverse_stranded` flags. Both are set as 
+`false` by default but reversed is cahnged to true in the Uppmax config file. If you library instead is forward oriented simply specify the`--forward_stranded` flag as `true`
```

Comment:
`cahnged` -> `changed`

I'm not sure that you need to do `--forward_stranded true`, I think you just do `--forward_stranded`